NewTutorials/Tutorial/PlayerViewControllerWrapper.swift

```diff
@@ -0,0 +1,57 @@
+//  Copyright © 2017 Oath. All rights reserved.
+
+import UIKit
+import OneMobileSDK
+import PlayerControls
+
+
+class PlayerViewControllerWrapper: UIViewController {
+    struct Props {
+        var isLoading = false
+        var player: Future<Result<Player>>?
+        var controlsViewController: ContentControlsViewController = DefaultControlsViewController()
+    }
+    
+    var props = Props() {
+        didSet {
+            view.setNeedsLayout()
+            
+            guard !props.isLoading else { return }
+            guard self.playerViewController?.player == nil else { return }
+            guard let player = props.player else { return }
```

Comment:
If we are passing `nil` here we should remove player from the `PlayerViewController` - otherwise it will stuck there.